### PR TITLE
Remove HLL, digests and backup/restore.

### PR DESCRIPTION
**Pull request by zeebo**
Updates #896.

### CODE REVIEW

**Comment by e-dard** [comment]
I'll review this on Monday.

File: tsdb/tsi1/index_file.go [hunk 1]
```diff
@@ -32,8 +30,7 @@ const (
 		8 + 8 + // measurement block offset + size
 		8 + 8 + // series id set offset + size
 		8 + 8 + // tombstone series id set offset + size
-		8 + 8 + // series sketch offset + size
-		8 + 8 + // tombstone series sketch offset + size
+		8 + 8 + 8 + 8 + // legacy sketch info
```
Comment (e-dard) [inline tsdb/tsi1/index_file.go]: Can we just add a little more to this comment? Maybe explaining we took out HLL?

**Comment by e-dard** [comment]
@benbjohnson I would like you to review this, with special attention on any problems you might forsee with opening existing tsi files with embedded sketches.

**Comment by benbjohnson** [comment]
@e-dard Looks like it's simply zero'ing out the header fields on write and skipping over them on read so I don't see any issues on reading old files.

**Comment by e-dard** [comment]
@zeebo 🎉

**Pull request merged**